src/html/Quoting.cc

```diff
@@ -9,50 +9,48 @@
 #include "squid.h"
 #include "html/Quoting.h"
 
-/*
- *  HTML defines these characters as special entities that should be quoted.
- */
-static struct {
-    unsigned char code;
-    const char *quote;
-} htmlstandardentities[] =
+#include <array>
+#include <cstring>
+
 
+static const auto& MakeEscapeSequences()
```

Comment:
With the current implementation, this is not a "maker" function (i.e. a function that gives you a new thing every time you call it). This is an accessor to escape sequences (that have complex initialization logic). Please rename it to EscapeSequences(), AllEscapeSequences(), or something like that _and_ either assert() that it is never called twice (simply because we do not want to spend time on supporting multiple calls) or adjust it a bit to return the previously initialized array on repeated calls. You can use `escapeMap` for that assert.